src/screens/NewsScreen/Content.js

```diff
@@ -59,7 +59,7 @@ const Content = (props) => {
               testID="textPress"
               onPress={() => Linking.openURL(sanitizeUrlForLinking(url))}
               style={{
-                color: COLORS.blue,
+                color: COLORS.signed_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from `COLORS.blue` to `COLORS.signed_primary`. Ensure that this color change aligns with the overall design and accessibility guidelines. High contrast between text and background is important for readability.

```diff
- color: COLORS.blue,
+ color: COLORS.signed_primary,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->